tox.ini

```diff
@@ -2,5 +2,5 @@
 envlist = py26, py27, pypy
 
 [testenv]
-deps = -rrequirements.txt
+deps = -rrequirements/dev.txt
```

Comment:
@pydanny `requirements.txt` is a symlink to `requirements/dev.txt`, which appveyor / windows didn't seem to like (https://ci.appveyor.com/project/michaeljoseph/changes/build/1.0.71/job/jddradg2h0rxeiqs)